setup.py

```diff
@@ -15,13 +15,13 @@
 	long_description_content_type="text/markdown",
 	packages=find_packages(),
 	classifiers=[
-        "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: OS Independent",
-    ],
+		"Programming Language :: Python :: 3",
+		"License :: OSI Approved :: MIT License",
+		"Operating System :: OS Independent",
+	],
 	entry_points={
 		'console_scripts': [
 			'hsc=hsc.crawler:main',
 		],
-	}
+	}, install_requires=['requests']
```

Comment:
We can also add **getpass** in this list.

Reply:
ok sure

Reply:
I think getpass is by default installed